### ISSUE TITLE
File export

### ISSUE THREAD

**Issue by eggplants** [nandenjin/twinkle-parser#31]
Now, I export formatted json like this:
```bash
$ yarn run parse kdb.csv|
sed -r '1,2d;s/(})[^}]+$/\1/'|
python -c 'import sys,json
print(
    json.dumps(
        json.loads(sys.stdin.read()),
        indent=4,
        ensure_ascii=False
    )
)' >export_kdb.json
```
I want to be more easy to export to a local file. Could I ask you to do that...?
Thanks.

**Comment by nandenjin** [comment]
I thought that `yarn run parse` is just a shortcut to tests and directly executed by `node dist/index.js` to export data. Your suggestion is LGTM so I'll implement something to useful to exporting. 

You can also use `node dist/index.js PATH_TO_SRC` until I implement it.
Thanks.

**Comment by eggplants** [comment]
Thanks!

**Comment by nandenjin** [comment]
Implemented and merged by #32 , and now available with v0.1.5. Thank you.
